torchvision/models/detection/keypoint_rcnn.py

```diff
@@ -253,10 +253,9 @@ def __init__(self, in_channels, num_keypoints):
 
     def forward(self, x):
         x = self.kps_score_lowres(x)
-        x = misc_nn_ops.interpolate(
-            x, scale_factor=float(self.up_scale), mode="bilinear", align_corners=False
+        return torch.nn.functional.interpolate(
```

Comment:
I've merged a PR that makes a conflict here, can you rebase?

Reply:
I rebased it, conflict solved